utils/python/tests/scripts_regression_tests.py

```diff
@@ -1771,6 +1778,26 @@ def _main_func():
 
     CIME.utils.handle_standard_logging_options(args)
 
+    # Find all python files in repo and create a pylint test for each
+    if check_for_pylint():
+        cimeroot = CIME.utils.get_cime_root()
+        files_to_test = run_cmd_no_fail("git ls-files --full-name %s"%cimeroot).splitlines()
+        files_to_test = [item for item in files_to_test if item.endswith(".py")]
+        files_to_test.extend(run_cmd_no_fail("git grep -l '#!/usr/bin/env python'",
+                                        from_dir=cimeroot).splitlines())
+        #TODO - get rid of this
+        list_of_directories_to_ignore = ("scripts/Tools", "point_clm", "tools", "machines", "apidocs", "unit_test")
+        for file_ in files_to_test:
```

Comment:
This makes each pylint file test a separate test in scripts_regression_tests and extends it to include the 
scripts directory and the buildnml and buildexe from driver and component models.   The list_of_directories_to_ignore is a todo list of files to update to conform to pylint requirements.